doc/adrs/XXX_add_table_description_in_view_dsl.adoc

```diff
@@ -0,0 +1,99 @@
+= ADR-XXX Add Table Description in View DSL
+
+== Context
+
+Add table description to View DSL.
+
+== Decision
+
+=== View DSL
+
+* *Name*: _view#Identifier_, name of the representation.
+* *Label Expression*: _AQL expression_, default name for an instance of this representation.
+* *Icon URLs Expression*: _AQL expression_, icons for this representation.
+* *Description Expression*: _AQL expression_, default description for an instance of this representation.
+* *Precondition Expression*: _AQL expression_, predicate to create representation.
+* *Domain Type*: _view#DomainType_, type of element on which a representation can be created.
+* *Is Stripe Row Semantic Elements Expression*: _AQL expression_, enable or disable background color fading on lines.
+* *Column Descriptions*: _containment multivalued_
```

Comment:
Please add some structure in this document instead of a massive list of features. Split things and where necessary add some relevant comments. Something like an [Xcore](https://wiki.eclipse.org/Xcore) like syntax would be way more understandable:

```
class TableDescription extends RepresentationDescription {
    InterpretedExpression useStripedRowsExpression
    contains ColumnDescription[] columnDescriptions
    // etc...
}
```

It would make it way easier to see what you are proposing and it would remove the need to repeat existing stuff like `RepresentationDescription#name` and prevent mistakes like `RepresentationDescription#titleExpression` vs the `labelExpression` that you have mentioned above.

Reply:
Please add a name to the column like `NodeDescription#name` to help document the DSL